### PR TITLE
Release/1.34.2

### PR DESCRIPTION
**Pull request by alexsanford**
* Update changelog.
* Bump version.
* Update POT file.

Package for testing: [wp-job-manager-1.34.2.zip](https://github.com/Automattic/WP-Job-Manager/files/4697046/wp-job-manager-1.34.2.zip)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ WP Job Manager is a **lightweight** job listing plugin for adding job-board like
 * Each listing can be tied to an email or website address so that job seekers can apply to the jobs.
 * Searches also display RSS links to allow job seekers to be alerted to new jobs matching their search.
 * Allow logged in employers to view, edit, mark filled, or delete their active job listings.
-* Developer friendly code — Custom Post Types, endpoints & template files.
+* Developer friendly code â Custom Post Types, endpoints & template files.
```
Comment (donnapep) [inline readme.md:26]: What's up with that funk character?

File: readme.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 **Requires at least:** 4.9  
 **Tested up to:** 5.2
```
Comment (donnapep) [inline readme.md:5]: 5.4?

File: readme.txt [hunk 3]
```diff
@@ -4,7 +4,7 @@ Tags: job manager, job listing, job board, job management, job lists, job list,
 Requires at least: 4.9
 Tested up to: 5.2
```
Comment (donnapep) [inline readme.txt:5]: 5.4?

File: readme.txt [hunk 4]
```diff
@@ -23,7 +23,7 @@ WP Job Manager is a **lightweight** job listing plugin for adding job-board like
 * Each listing can be tied to an email or website address so that job seekers can apply to the jobs.
 * Searches also display RSS links to allow job seekers to be alerted to new jobs matching their search.
 * Allow logged in employers to view, edit, mark filled, or delete their active job listings.
-* Developer friendly code — Custom Post Types, endpoints & template files.
+* Developer friendly code â Custom Post Types, endpoints & template files.
```
Comment (donnapep) [inline readme.txt:26]: Ditto here.

File: wp-job-manager.php [hunk 5]
```diff
@@ -3,7 +3,7 @@
  * Plugin Name: WP Job Manager
  * Plugin URI: https://wpjobmanager.com/
  * Description: Manage job listings from the WordPress admin panel, and allow users to post jobs directly to your site.
- * Version: 1.34.1
+ * Version: 1.34.2
```
Comment (donnapep) [inline wp-job-manager.php:6]: There's a "Tested up to" in this file as well.

File: readme.md (hunk 1 above)
Comment (alexsanford) [reply readme.md]: Fantastic question. Fixed in https://github.com/Automattic/WP-Job-Manager/pull/2018/commits/edae4a1176362a5bc7ce56d7dabdb6099783391b

File: readme.md (hunk 2 above)
Comment (alexsanford) [reply readme.md]: https://github.com/Automattic/WP-Job-Manager/pull/2018/commits/55dce87944b37f94da716a9a2b01ae3b73617b86

File: readme.txt (hunk 3 above)
Comment (alexsanford) [reply readme.txt]: https://github.com/Automattic/WP-Job-Manager/pull/2018/commits/55dce87944b37f94da716a9a2b01ae3b73617b86

File: wp-job-manager.php (hunk 5 above)
Comment (alexsanford) [reply wp-job-manager.php:6]: https://github.com/Automattic/WP-Job-Manager/pull/2018/commits/55dce87944b37f94da716a9a2b01ae3b73617b86

File: readme.txt (hunk 4 above)
Comment (alexsanford) [reply readme.txt]: https://github.com/Automattic/WP-Job-Manager/commit/edae4a1176362a5bc7ce56d7dabdb6099783391b

**Pull request merged**